packages/inventory-compliance/src/SystemPolicyCard.js

```diff
@@ -81,15 +81,19 @@ class SystemPolicyCard extends React.Component {
                         </Tooltip> }
                     </TextContent>
                     <div className='margin-bottom-md' >
-                        <Tooltip content={
-                            'The system compliance score is calculated by OpenSCAP and ' +
-                            'is a normalized weighted sum of rules selected for this policy.'
-                        }>
-                            { this.complianceIcon(compliant) }
-                            <Text component={ TextVariants.small }>
-                                { rulesPassed } of { rulesPassed + rulesFailed } rules passed ({ passedPercentage })
-                            </Text>
-                        </Tooltip>
+                        { this.complianceIcon(compliant) }
+                        <Text component={ TextVariants.small }>
+                            { rulesFailed } rule{ rulesFailed === 1 ? '' : 's' } failed
+                            { " " }
```

Comment:
This is not needed. Just use a space within the `span` It should be preserved and not ignored when rendering.

Reply:
If you mean `<span> </span>`, yes. I wanted to avoid adding the space into the span that has the tooltip. `{ " " }` was one of the (shorter) options mentioned on forums.

Reply:
Shorter than a ` `?

Reply:
It doesn't hurt if it is also triggered on that space. Makes the target bigger. ;)

Reply:
Here are the options:
* `{ " " }`
* `<span> </span>`
* `&nbsp;` (not sure if we want to have a non-breaking space)

A space in the span down below in the tooltip is not what I wanted to do. It changes the tooltip positioning as well as the trigger (if we want to be precise).

Reply:
I'd prefer just adding a space to the span at the bottom.
It doesn't need to be precise. And it only marginally, about 1,5px, skews the position.

Behaviour around preserving space in JSX has been changing over time so I'd avoid it.

Reply:
I would not worry about the bracket notation with whitespace. If JSX would break it, then majority of notations, not related to whitespace, will break with it.